src/compile.c

```diff
@@ -577,7 +578,7 @@ block gen_param(const char* name) {
 }
 
 block gen_lambda(block body) {
-  return gen_function("@lambda", gen_noop(), body);
+  return gen_function("#lambda", gen_noop(), body);
```

Comment:
Did the "@" have something do the current `@format` code? seeing that `gen_lambda` it used for non-format things also. Was incorrect?

Reply:
I believe this has nothing to do with string format, it's simply a randomly chosen symbol. But now string formats are now treated like functions, `@lambda` is a bit confusing, especially when shown in the debug traces.